pandas/core/series.py

```diff
@@ -3979,7 +3979,9 @@ def transform(
     ) -> FrameOrSeriesUnion:
         return transform(self, func, axis, *args, **kwargs)
 
-    def apply(self, func, convert_dtype=True, args=(), **kwds):
+    def apply(
+        self, func: AggFuncType, convert_dtype: bool = True, args: tuple = (), **kwds
```

Comment:
use Tuple from typing and add type parameters where possible

```suggestion
        self, func: AggFuncType, convert_dtype: bool = True, args: Tuple[Any] = (), **kwds
```